tests/src/test/scala/com/nvidia/spark/rapids/GpuCoalesceBatchesSuite.scala

```diff
@@ -127,29 +128,36 @@ class GpuCoalesceBatchesSuite extends SparkQueryCompareTestSuite {
 
     withGpuSparkSession(spark => {
 
+      ExecutionPlanCaptureCallback.startCapture()
+
       val df = longsCsvDf(spark)
 
       // currently, GpuSortExec requires a single batch but this is likely to change in the
       // future, making this test invalid
       val df2 = df
         .sort(df.col("longs"))
 
-      val coalesce = df2.queryExecution.executedPlan
+      // execute the plan
+      df2.collect()
+
+      val executedPlan = ExecutionPlanCaptureCallback.getResultWithTimeout().get match {
```

Comment:
Matching should unwrap the `get` here, e.g.:
```scala
val executedPlan = ExecutionPlanCaptureCallback.getResultWithTimeout() match {
  Some(a: AdaptiveSparkPlanExec) => a.executedPlan
  Some(other) => other
  None => ...throw or something here...
```
Applies to other places where `getResultWithTimeout().get` is used.

Reply:
These are updated now.